dev-lua/lua-bit32/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+<email>azamat.hackimov@gmail.com</email>
+<name>Azamat H. Hackimov</name>
+</maintainer>
+<maintainer type="project">
+<email>proxy-maint@gentoo.org</email>
+<name>Proxy Maintainers</name>
+</maintainer>
+<upstream>
+<remote-id type="github">keplerproject/lua-compat-5.3</remote-id>
+</upstream>
+</pkgmetadata>
```

Comment:
Could you indent these better? 
https://github.com/gentoo/gentoo/blob/master/skel.metadata.xml